misc/python/materialize/mzcompose/composition.py

```diff
@@ -788,7 +788,6 @@ def run(
             capture_and_print=capture_and_print,
             stdin=stdin,
             check=check,
-            environment=env_extra,
```

Comment:
That alone won't work, it needs this change above:
```
            *(f"-e{k}={v}" for k, v in env_extra.items()),
```

But that will produce very long log messages :/

Reply:
Oh, right. We can filter them out when printing, or pass in the original environment too. I'll do the latter for now.